Error if no openvex output

## The Issue
The action raises an error in the event that *Trivy* does not find fixable vulnerabilities and *Copa* does not generate the Openvex output.

## How to Reproduce

Test the `patch` action with the image `docker.io/library/python:3.11.7-slim-bookworm`, in this case the Copa Action fails with the following error:  


```console
chmod: cannot access './data/docker.io-library-python-3.11.7-slim-bookworm-openvex-report.json': No such file or directory
Error: Process completed with exit code 1.
```

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -29,9 +29,3 @@ else
     exit 1
 fi
 
-# due to docker, file permissions are set as 400
-# set output file permissions so user can read it
-if [ ! -z "$output_file" ]
-then
-    chmod 444 ./data/"$output_file"
-fi

